client/src/store/store.ts

```diff
@@ -18,14 +19,41 @@ const rootReducer = combineReducers({
   files: fileSlice,
   cart: cartSlice,
   libraryConfigFiles: libraryConfigFilesSlice,
+  executionHistory: executionHistorySlice,
 });
 
 const store = configureStore({
   reducer: rootReducer,
   middleware: (getDefaultMiddleware) =>
     getDefaultMiddleware({
       serializableCheck: {
-        ignoredActions: ['digitalTwin/setDigitalTwin'],
+        ignoredActions: [
+          // Digital Twin actions that contain class instances
+          'digitalTwin/setDigitalTwin',
+          'digitalTwin/setJobLogs',
+          'digitalTwin/setPipelineCompleted',
+          'digitalTwin/setPipelineLoading',
+          'digitalTwin/setShouldFetchDigitalTwins',
+          // Asset actions that contain LibraryAsset class instances
+          'assets/setAssets',
+          'assets/setAsset',
+          'assets/deleteAsset',
+          // Execution history actions
+          'executionHistory/addExecutionHistoryEntry',
```

Comment:
can this part be simplified but refactoring the `executionHistory.slice` and `digitaltwin.slice`?